.github/workflows/release.yaml

```diff
@@ -32,40 +52,26 @@ jobs:
           IMAGE: quay.io/kairos/framework
         run: |
           export TAG=${GITHUB_REF##*/}
-          ./earthly.sh +framework-image --IMG=${IMAGE}:$TAG --FLAVOR=${FLAVOR} --WITH_KERNEL=false
-          ./earthly.sh +framework-image --IMG=${IMAGE}-generic:$TAG --FLAVOR=${FLAVOR} --WITH_KERNEL=true
+          earthly --ci --push +build-framework-image --FLAVOR=${FLAVOR}
```

Comment:
should this push every single time or only on master?

Reply:
ok should be good now, the `release.yaml` pushes always (as it is triggered on tags from branches), while the `image.yaml` pushes only on master